web/viewer.css

```diff
@@ -308,6 +316,26 @@ select {
   right: 0;
   z-index: 9999;
   cursor: default;
+  font: message-box;
+}
+
+.toolbar input,
+.toolbar button,
+.toolbar select,
+.secondaryToolbar input,
+.secondaryToolbar button,
+.secondaryToolbar select,
+.editorParamsToolbar input,
+.editorParamsToolbar button,
+.editorParamsToolbar select,
+.findbar input,
+.findbar button,
+.findbar select,
+#sidebarContainer input,
+#sidebarContainer button,
+#sidebarContainer select {
```

Comment:
I believe that you'll need to add a `font: message-box;` rule here as well, since as-is some UI elements (e.g. the pageNumber-input and the zoom-dropdown) no longer have the correct fonts in the viewer.